src/plugins/vis_type_timeseries/public/application/components/color_picker.tsx

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
+// The color picker is not yet accessible.
+
+import React, { useState } from 'react';
+import { EuiIconTip, EuiColorPicker, EuiColorPickerSwatch } from '@elastic/eui';
+// @ts-ignore
+import { useColorPickerState, EuiColorPickerOutput } from '@elastic/eui/lib/services';
+import { i18n } from '@kbn/i18n';
+import { COLOR_SWATCHES } from './custom_color_swatches';
+
+const COMMAS_NUMS_ONLY_RE = /[^0-9,]/g;
+
+interface ColorProps {
+  [key: string]: string | null;
+}
+
+export interface ColorPickerProps {
+  name: string;
+  value: string | null;
```

Comment:
nit: I see in case of invalid value ColorPicker returns empty string. Maybe we can do something with `null` in interface? I'm not a big fan of cluttering up the interface. 

The same valid for ColorProps interface

Reply:
The problem here is that all these fields are `stringOptionalNullable` so it requires more changes to the mechanism that these values are set. So I would suggest to not do it here